doc/varnish-configuration.rst

```diff
@@ -4,7 +4,7 @@ Varnish Configuration
 ---------------------
 
 Below you will find detailed Varnish configuration recommendations for the
-features provided by this library. The examples are tested with Varnish
+features provided by this library. These examples have been tested with Varnish
```

Comment:
not sure, the examples actually *are* constantly tested on travis-ci... but this is too much detail here, maybe we can just say that we provide configuration for version 3 and 4, without talking about testing at all. wdyt?